ccpp/data/GFS_typedefs.meta

```diff
@@ -6971,6 +7037,81 @@
   dimensions = (horizontal_loop_extent)
   type = real
   kind = kind_phys
+[skebu_wts]
```

Comment:
There is no flipping of variables in the code in `stochastic_physics_wrapper.F90`

Reply:
I'm not sure where this crept into the code.  I think they're legacy entries from a while ago.  Thanks for finding them!  Removed.

Reply:
Removed.